### PR TITLE
Tab panel jaws subtext

### PR DESCRIPTION
**Pull request by Nayshlok**
Minor tweaks to help JAWS to read what needs to be read. The biggest change is the tab panels takes in what needs to be read from the parent.

### CODE REVIEW

**Comment by Nayshlok** [comment]
@KStClaire @aabenoja can I get this checked out? I tested it in the ui components and everything worked well.

File: packages/es-components/src/components/containers/tabPanels/Tab.js [hunk 1]
```diff
@@ -44,7 +45,7 @@ const TabButton = styled.button`
   }
 `;
 
-const AriaAnnouncer = styled.span`
+const AriaAnnouncer = styled.p`
   position: fixed;
```
Comment (Darrken) [inline packages/es-components/src/components/containers/tabPanels/Tab.js]: I've seen a couple of different methods used lately in our code to hide screenreader content, and maybe we should pick one and stick to it. [WebAIM](https://webaim.org/techniques/css/invisiblecontent/) has a good article on this and has a good (similar) solution. See the "Positioning Content Off-screen" section of that article. They add a few more rules to make sure there aren't any visual artifacts or other issues with this method.

Comment (Nayshlok) [reply packages/es-components/src/components/containers/tabPanels/Tab.js]: I can add those extra properties

**Comment by Nayshlok** [comment]
@Darrken Do you have any idea why the CI would have failed. When I looked it seemed to have hung for 10 minutes

**Pull request merged**